aspnetcore/security/authentication/community.md

```diff
@@ -13,12 +13,12 @@ This page contains community-provided, open source authentication options for AS
 
 ## OSS authentication providers
 
-The list below is sorted alphabetically.
-
 | Name | Description |
 | ---- | ----------- |
+| [IdentityServer](https://duendesoftware.com/products/identityserver) | IdentityServer is an OpenID Connect and OAuth 2.0 framework for ASP.NET Core. |
 | [Gluu Server](https://gluu.org/) | Enterprise ready, open source software for identity, access management (IAM), and single sign-on (SSO). For more information, see the [Gluu Product Documentation](https://gluu.org/docs/). |
-| [IdentityServer](https://identityserver.io/) | IdentityServer is an OpenID Connect and OAuth 2.0 framework for ASP.NET Core, officially certified by the OpenID Foundation and under governance of the .NET Foundation. For more information, see [Welcome to IdentityServer4 (Documentation)](https://identityserver4.readthedocs.io/en/latest/). |
 | [OpenIddict](https://github.com/openiddict/openiddict-core) | Versatile OpenID Connect stack for ASP.NET Core 2.1/3.1/5.0 and Microsoft.Owin 4.1 (compatible with ASP.NET 4.6.1). |
+| [AspNet.Security.OAuth.Providers](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers)  | A collection of security middleware for ASP.NET Core apps to support social authentication  |
+| [AspNet.Security.OpenId.Providers](https://github.com/aspnet-contrib/AspNet.Security.OpenId.Providers) | A collection of security middleware for ASP.NET Core apps to support OpenID 2.0 authentication providers like [Steam](https://steampowered.com/), [Wargaming](https://wargaming.net/), and [Orange](https://www.orange.fr/). |
```

Comment:
FWIW, Orange no longer supports OpenID 2.0 (they removed their 2.0 endpoints last year and moved to OIDC). I updated the OpenID repo to remove Orange from the sample.